west.yml

```
@@ -198,7 +198,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 466000a80e6eb5bbcb691bae936ec5654a7796d4
+      revision: 7a17a78b14565a12a6dc5e527f71c351ed4df6b6
```

```suggestion
      revision: pull/433/head
```
Don't grab the new SHA directly, that won't be possible until the PR in the HAL merges. Instead, use syntax like I suggested above so west can pull in the contents of the PR